server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -308,7 +308,12 @@ private void onlyCreateIndex(
         final CreateIndexClusterStateUpdateRequest request,
         final ActionListener<AcknowledgedResponse> listener
     ) {
-        normalizeRequestSetting(request);
+        try {
```

Comment:
Is this change still required?

Reply:
Yes, since validation can throw exceptions, and the caller does not expect them from this method.